src/theory/bv/theory_bv_rewriter.cpp

```diff
@@ -59,12 +59,13 @@ RewriteResponse TheoryBVRewriter::RewriteBitOf(TNode node, bool prerewrite)
 
 RewriteResponse TheoryBVRewriter::RewriteUlt(TNode node, bool prerewrite) {
   // reduce common subexpressions on both sides
-  Node resultNode = LinearRewriteStrategy
-    < RewriteRule<EvalUlt>, // if both arguments are constants evaluates
-      RewriteRule<UltZero>, // a < 0 rewrites to false,
-      RewriteRule<SignExtendUltConst>,
-      RewriteRule<ZeroExtendUltConst>
-       >::apply(node);
+  Node resultNode =
+      LinearRewriteStrategy<RewriteRule<EvalUlt>,  // if both arguments are
```

Comment:
I suggest removing the comments from here:
1. We don't have it in `RewriteUle`
2. It is better to document the rewrite in the corresponding h/cpp file where the rewrite is declared / implemented.